source/features/show-associated-branch-prs-on-fork.tsx

```diff
@@ -75,6 +76,29 @@ function addAssociatedPRLabel(branchCompareLink: Element, prInfo: PullRequest):
 	);
 }
 
+function addAssociatedPRLabelNew(parent: Element, prInfo: PullRequest): void {
+	const StateIcon = stateIcon[prInfo.state];
+
+	parent.append(
+		<div className="rgh-pr-box">
+			<a
+				href={prInfo.url}
+				target="_blank"
+				data-hovercard-url={prInfo.url + '/hovercard'}
+				aria-label={`Link to the ${prInfo.isDraft ? 'draft ' : ''}pull request #${prInfo.number}`}
+				className="rgh-pr-link" rel="noreferrer"
+			>
+				<span className="rgh-pr-label">
+					<div data-testid="draft-pull-request-icon" className="rgh-pr-box">
+						<StateIcon width={14} height={14} className={'rgh-pr-' + prInfo.state.toLowerCase()}/>
```

Comment:
GitHub already has dedicated color classes:

```
.draft.octicon {
    color: var(--fgColor-muted, var(--color-fg-muted))
}

.closed.octicon,.reverted.octicon {
    color: var(--fgColor-closed, var(--color-closed-fg))
}

.open.octicon {
    color: var(--fgColor-open, var(--color-open-fg))
}

.closed.octicon.octicon-issue-closed,.merged.octicon {
    color: var(--fgColor-done, var(--color-done-fg))
}
```

so I think this will already work:

```suggestion
						<StateIcon width={14} height={14} className={prInfo.state.toLowerCase()}/>
```

plus some adjustments for `draft`